frontend/src/components/eam/eam.tsx

```diff
@@ -1,17 +1,34 @@
 import { FC } from 'react';
 import { Button, IconButton } from 'components/common/common';
 import { AppRoute, IconName } from 'common/enums/enums';
-import { useAppDispatch, useAppSelector, useEffect } from 'hooks/hooks';
+import {
+  useAppDispatch,
+  useAppSelector,
+  useEffect,
+  useState,
+} from 'hooks/hooks';
 import { eam as eamActions } from 'store/actions';
 import { GroupsTable, WorkersTable, Tenant } from './components/components';
 import styles from './styles.module.scss';
+import {
+  WORKERS_PER_PAGE,
+  WORKER_BACK_PAGE,
+  WORKER_NEXT_PAGE,
+  WORKER_CURRENT_PAGE,
+} from 'common/constants/workers.constants';
 
 const EAM: FC = () => {
   const dispatch = useAppDispatch();
+  const [currentPage, setCurrentPage] = useState(1);
+  const [from, setFrom] = useState(0);
 
-  const { id: tenantId } = useAppSelector(({ app }) => ({
-    id: app.tenant?.id,
-  }));
+  const { id: tenantId, countItems: countItems } = useAppSelector(
+    ({ app, eam }) => ({
+      id: app.tenant?.id,
+      countItems: eam.countItems,
+    }),
+  );
```

Comment:
I think its not necessarily.
```suggestion
  const { tenantId, countItems: countItems } = useAppSelector(
    ({ app, eam }) => ({
      tenantId: app.tenant?.id,
      countItems: eam.countItems,
    }),
  );
```

Reply:
This is not from my pull request and assignment, not sure if it needs to be edited here

Reply:
Here added id property what immediately overwritten in tenantId, I think we don't need it.
Wait what the coaches say.

Reply:
```suggestion
  const { id: tenantId, countItems } = useAppSelector(
    ({ app, eam }) => ({
      id: app.tenant?.id,
      countItems: eam.countItems,
    }),
  );
```